packages/jupyter-ai-magics/jupyter_ai_magics/providers.py

```diff
@@ -596,14 +614,41 @@ class BedrockProvider(BaseProvider, Bedrock):
     id = "bedrock"
     name = "Amazon Bedrock"
     models = [
-        "amazon.titan-tg1-large",
+        "amazon.titan-text-express-v1",
         "anthropic.claude-v1",
+        "anthropic.claude-v2",
         "anthropic.claude-instant-v1",
+        "ai21.j2-ultra-v1",
+        "ai21.j2-mid-v1",
+        "cohere.command-text-v14",
+    ]
+    model_id_key = "model_id"
+    pypi_package_deps = ["boto3"]
+    auth_strategy = AwsAuthStrategy()
+    fields = [
+        TextField(
+            key="credentials_profile_name",
+            label="AWS profile (optional)",
+            format="text",
+        ),
+        TextField(key="region_name", label="Region name (optional)", format="text"),
```

Comment:
Is region name intended to be optional here? This is not the case for the SM Endpoints provider.

Reply:
`region_name` should be optional, even for SM endpoints.